ultralytics/solutions/object_counter.py

```diff
@@ -230,6 +250,23 @@ def extract_and_process_tracks(self, tracks):
                 color=self.count_color,
             )
 
+    """ Returns the values of the corresponding object"""
+
+    def get_incoming(self):
+        return self.incoming
+
+    def get_outgoing(self):
+        return self.outgoing
+
+    def get_in_counts(self):
+        return self.in_counts
+
+    def get_out_counts(self):
+        return self.out_counts
+
+    def get_two_way(self):
+        return self.two_way
```

Comment:
Make sure to add docstrings. Would be nice to add return annotations, for example `def get_incoming(self) -> dict[int]:`